packages/js/uri-resolver-extensions/src/UriResolverExtensionFileReader.ts

```diff
@@ -18,7 +18,7 @@ export class UriResolverExtensionFileReader implements IFileReader {
   ) {}
 
   async readFile(filePath: string): Promise<Result<Uint8Array, Error>> {
-    return await UriResolverInterface.module.getFile(
+    const result = await UriResolverInterface.module.getFile(
```

Comment:
We should change the signature of the getFile method here to return `Promise<Result<Uint8Array | undefined, Error>>` instead of `Promise<Result<Uint8Array, Error>>`.
This mimicks the URI resolver interface signature

Reply:
Should we do this in a separate PR @nerfZael?

Reply:
I think it can be a part of this PR, since it's a one line change that doesn't impact much and most importantly, it addresses the line below `if (!result.value)` which doesn't make sense without it since according to the signature, result.value can never be falsy.

Reply:
Ah okay I see what you meant, the typings for the `UriResolverInterface` are incorrect, fixed!